lib/spoom/coverage/snapshot.rb

```diff
@@ -22,6 +22,8 @@ class Snapshot < T::Struct
       prop :calls_untyped, Integer, default: 0
       prop :calls_typed, Integer, default: 0
       prop :sigils, T::Hash[String, Integer], default: Hash.new(0)
+      prop :methods_with_sig_excluding_rbis, Integer, default: 0
+      prop :non_generated_sigils, T::Hash[String, Integer], default: Hash.new(0)
```

Comment:
```suggestion
      prop :sigils_excluding_rbis, T::Hash[String, Integer], default: Hash.new(0)
```

Reply:
It might confuse people because we aren't excluding all `.rbi` files but I don't have strong opinions.